Framework/API/inc/MantidAPI/ISpectrum.h

```diff
@@ -1,13 +1,17 @@
 #ifndef MANTID_API_ISPECTRUM_H_
 #define MANTID_API_ISPECTRUM_H_
 
-#include "MantidKernel/System.h"
 #include "MantidGeometry/IDTypes.h"
+#include "MantidHistogramData/Histogram.h"
+#include "MantidKernel/System.h"
 #include "MantidKernel/cow_ptr.h"
 
 #include <set>
 
 namespace Mantid {
+namespace DataObjects {
```

Comment:
Should we be forward declaring across modules like this? I can't see where it is used.